test/wpt/status/streams.json

```diff
@@ -54,10 +54,44 @@
       ]
     }
   },
+  "transform-streams/cancel.any.js": {
+    "fail": {
+      "expected": [
+        "cancelling the readable side should call transformer.cancel()",
+        "cancelling the readable side should reject if transformer.cancel() throws",
+        "aborting the writable side should call transformer.abort()",
+        "aborting the writable side should reject if transformer.cancel() throws",
+        "closing the writable side should reject if a parallel transformer.cancel() throws",
+        "readable.cancel() and a parallel writable.close() should reject if a transformer.cancel() calls controller.error()",
+        "writable.abort() and readable.cancel() should reject if a transformer.cancel() calls controller.error()"
+      ]
+    }
+  },
+  "transform-streams/errors.any.js": {
+    "fail": {
+      "expected": [
+        "controller.error() should close writable immediately after readable.cancel()"
+      ]
+    }
+  },
+  "transform-streams/general.any.js": {
+    "fail": {
+      "expected": [
+        "terminate() should abort writable immediately after readable.cancel()"
+      ]
+    }
+  },
```

Comment:
These new tests are for `transformer.cancel()`, which will hopefully be implemented soon by https://github.com/nodejs/node/pull/50126.